articles/api-auth/token-introspection/index.md

```diff
@@ -0,0 +1,87 @@
+---
+description: How to use token introspection to get information about a token's authorization context
+url: /api-auth/token-introspection
+toc: true
+---
+
+# Token Introspection
+
+In OAuth 2.0, the clients don't need to know anything about the content or structure of the token. However, there may be metadata attached to the token the client would find useful, such as the token's validity, the approved scopes, and the context in which the token was issued.
+
+Protected resources, such as APIs, can use token metadata to make authorization decisions. One protocol that allows protected resources to query an authorization server to obtain the set of metadata for a given token is called *token introspection*.
+
+Auth0 supports [OAuth 2.0 Token Introspection](https://tools.ietf.org/html/rfc7662), which allows a protected resource to query Auth0 to request the metadata attached to an OAuth 2.0 token. This is especially useful for [Application Specific Passwords (ASPs)](/api-auth/token-introspection/app-specific-passwords), which utilizes non-JWT tokens.
+
+::: note Because token instrospection is a means by which an API can verify a token, you do not need to configure any additional verification flows performed by the API. Additionally, for any existing authorization flows involving JWT access tokens, you do not need to update your processes to use token introspection.
+:::
+
+## Token Types
+
+You can use token introspection to request information for [**JWT access** tokens](/tokens/access-token), non-JWT access tokens, and [**refresh** tokens](/tokens/preview/refresh-token).
+
+## Create an API Record with Auth0
+
+To use token introspection, you must set up an [API](/apis) that represents your protected resource(s).
+
+For assistance, please see [How to Configure an API in Auth0](/apis#how-to-configure-an-api-in-auth0). Alternatively, you can do this by making the appropriate `POST` call to the Management API's [Create a Resource Server endpoint](/api/management/v2#!/Resource_Servers/post_resource_servers).
+
+### Provide the Public Key to Auth0
+
+Because token introspection requires Auth0 to validate the signature of your access token, you'll need to create a key pair for your API and update your API's Auth0 record with the public key.
```

Comment:
Something like this instead:

Token introspection uses the jwt-bearer profile for authentication (https://tools.ietf.org/html/rfc7523). Because Auth0 needs to validate the signature of this jwt, you need to update your API's Auth0 record with the public key.